actionview/lib/action_view/helpers/tags/file_field.rb

```diff
@@ -2,6 +2,17 @@ module ActionView
   module Helpers
     module Tags # :nodoc:
       class FileField < TextField # :nodoc:
+
+        def render
+          options = @options.stringify_keys
+          add_default_name_and_id(options)
+          if options.fetch("include_hidden", true)
+            tag("input", :name => options["name"], :type => "hidden", :value => "") + super
+          else
+            @options.delete(:include_hidden)
```

Comment:
This may fail if people set use string as key. I think it is better to delete before the conditional and here just do `@options = options`.